classes/dbutils.js

```diff
@@ -52,6 +58,7 @@ async function update_matches_stats_cache(db, match_id, is_network1_win) {
     } else {
         match.network1_losses += 1;
     }
+    _update_winrate(matches.slice(0, 1));
```

Comment:
The match being updated isn't always the first match. Probably could just pass in `[match]`

Reply:
Good catch. Fixed.